src/cata_tiles.cpp

```diff
@@ -416,8 +416,8 @@ int cata_tiles::load_tileset(std::string img_path, int R, int G, int B, int spri
 }
 
 void cata_tiles::set_draw_scale(int scale) {
-    tile_width = default_tile_width * scale / 16;
-    tile_height = default_tile_height * scale / 16;
+    tile_width = default_tile_width * tile_pixelscale * scale / 16;
+    tile_height = default_tile_height * tile_pixelscale * scale / 16;
```

Comment:
We're relying on SDL to do the scaling for us during blitting, as is already done with zooming?